JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/profile/AboutSection.kt

```diff
@@ -16,23 +16,33 @@
 
 package com.google.jetstream.presentation.screens.profile
 
+import android.content.Context
+import android.content.pm.PackageManager
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
+import androidx.tv.material3.ExperimentalTvMaterial3Api
 import androidx.tv.material3.MaterialTheme
 import androidx.tv.material3.Text
-import com.google.jetstream.BuildConfig
 import com.google.jetstream.data.util.StringConstants
 
+@OptIn(ExperimentalTvMaterial3Api::class)
 @Composable
 fun AboutSection() {
+    val context = LocalContext.current
+    val versionNumber = remember(context) {
+        getVersionNumber(context)
+    }
```

Comment:
NIT: This actually seems like a static data that `BuildConfig` might be good for. There's no need to resolve this at runtime (if there isn't something dynamically retrieved).

Reply:
Thanks! I realized that I accidentally removed the setting to generate BuildConfig from build.gradle.kts. I updated my PR to refer to BuildConfig to display version number.